test/DynamicDependency/data/Microsoft.WindowsAppRuntime.Framework/appxmanifest.xml

```diff
@@ -142,6 +142,12 @@
         <ActivatableClass ActivatableClassId="Microsoft.Windows.System.Power.PowerManager" ThreadingModel="both" />
       </InProcessServer>
     </Extension>
+    <Extension Category="windows.activatableClass.inProcessServer">
```

Comment:
Move this block up and keep in sorted order

Reply:
Done.